scanning/disa-alignment/ansible/test.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python3
+import os
+import re
+import subprocess
+
+from lib import util, results, virt, versions, ansible
+from conf import partitions, remediation
+
+
+ansible.install_deps()
+virt.Host.setup()
+
+profile = 'xccdf_org.ssgproject.content_profile_stig'
```

Comment:
I would do
```python
profile = 'stig'
profile_full = f'xccdf_org.ssgproject.content_profile_{profile}'
```
like in other Ansible tests, because then you can do ...